scipy/stats/__init__.py

```diff
@@ -432,7 +433,17 @@
    PearsonRNearConstantInputWarning
    SpearmanRConstantInputWarning
 
+
+Classes returned by some functions
+==================================
+
+.. autosummary::
+   :toctree: generated/
```

Comment:
```suggestion
   :toctree: generated/
   :hidden:
```

Reply:
>One thing to do before merging is fix the docs as Ralf suggested.

Not sure I follow what we want to achieve w.r.t. the docs @mdhaber does this match? 

IIUC, that will ***hide*** the `scipy.stats._result_classes.RelativeRiskResult` entry from the TOC in sphinx.

Reply:
Yes, I think so. We would also want to remove the heading of this new section, which would no longer exist.